src/shared/hooks/useAccessDataForTable.tsx

```diff
@@ -360,6 +386,81 @@ const accessData = async (
   return { ...result, headerProperties, tableResource };
 };
 
+const resourceToLocalStorageResource = (resource: Resource): TDataSource => {
+  let localStorageRow: TDataSource;
+  try {
+    const resourceName = resource.name ?? resource['@id'] ?? resource._self;
+    localStorageRow = {
+      key: resource._self,
+      _self: resource._self,
+      id: resource['@id'],
+      name: resourceName,
+      project: resource._project,
+      description: resource.description ?? '',
+      createdAt: resource._createdAt,
+      updatedAt: resource._updatedAt,
+      source: 'studios',
+      type: isArray(resource['@type'])
+        ? resource['@type']
+        : [resource['@type'] ?? ''],
+
+      distribution: {
+        contentSize: isArray(resource.distribution)
+          ? sumBy(
+              resource.distribution,
+              distItem => distItem.contentSize?.value ?? 0
+            )
+          : resource.distribution?.contentSize ?? 0,
+        encodingFormat: isArray(resource.distribution)
+          ? resource.distribution.find(distItem => distItem.encodingFormat)
+              ?.encodingFormat ?? ''
+          : resource.distribution?.encodingFormat ?? '',
+        label: isArray(resource.distribution)
+          ? resource.distribution.map(distItem => {
+              const itemLabel =
+                distItem.label ??
+                `${resourceName}.${fileExtensionFromResourceEncoding(
+                  distItem.encodingFormat,
+                  'txt'
+                )}`;
+              return itemLabel;
+            })
+          : resource.distribution?.label ??
+            `${resourceName}.${fileExtensionFromResourceEncoding(
+              resource.distribution?.encodingFormat,
+              'txt'
+            )}`,
+        hasDistribution: has(resource, 'distribution'),
+      },
+    };
+  } catch (err) {
+    Sentry.captureException(err, {
+      extra: {
```

Comment:
why do you think about adding the user who request the data download,
it will help if we need more investigation
```
User {
    id?: string;
    ip_address?: string;
    email?: string;
    username?: string;
}
```

Reply:
you can get the user object from the global state

Reply:
I actually do like the idea of adding more information to our logs, but in this instance I don't find user's information useful in debugging. Here I am only logging an error when a resource could not be serialized. Since this issue is not user specific - i.e. any user would get the same error if they tried to add this resource to cart, I don't think logging user details is pertinent here.

Also, since it also contains username and email, I think it's also not secure to log it without anonymizing. 

I guess, what we should do though at some point is add more generic information (like route, perhaps project or org details) to all our sentry logs. In many cases, user information could also be added after anonymizing it though.